condicionales/comparatotAnos-2-6.js

```diff
@@ -0,0 +1,20 @@
+// Escriba un programa que pida un año y que escriba si es bisiesto o no.Se recuerda que los años bisiestos son múltiplos de 4, pero los múltiplos de 100 no lo son, aunque los múltiplos de 400 sí.
+const prompt = require("prompt-sync")();
+let year;
+
+year = prompt("Write a year and I will tell you if it is leap: ");
+if (year % 400 === 0) {
```

Comment:
in javascript is possible to divide a string with a number but is better to have a number for those operations instead a string